.stylelintrc

```diff
@@ -11,6 +11,7 @@
       "border-bottom-left-radius",
       "transition"
     ],
+    "scss/dollar-variable-default": null
```

Comment:
One _very minor_ nitpick: I noticed that in the "upstream" TWBS stylelit config has the scss/dollar-variable-default rule last (after the function-blacklist rule):
https://github.com/twbs/bootstrap/blob/master/.stylelintrc

So maybe it would make sense to keep ours in that order, too?  (Just to make it easy to track future upstream changes)